gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -45,15 +56,28 @@ def self.declare_sig(mod, arg, &blk)
       raise "Invalid argument to `sig`: #{arg}"
     end
 
-    loc = caller_locations(2, 1).first
+    loc = caller_locations(caller_depth + 1, 1).first
 
-    T::Private::DeclState.current.active_declaration = DeclarationBlock.new(mod, loc, blk, arg == :final)
+    DeclarationBlock.new(mod, loc, blk, arg == :final, raw)
+  end
 
-    nil
+  def self.__with_declared_signature(mod, declblock, &blk)
```

Comment:
We use `_` for private-but-needs-to-be-public methods elsewhere in this file. Is it worth sticking to that convention in the handful of changes this PR makes?

Reply:
Sure, I can do that.